.github/workflows/build-docker-cache-artifacts.yml

```diff
@@ -0,0 +1,67 @@
+name: Docker Build Cache Save Artifact
+
+on:
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - 'docs/**'
+      - '**.md**'
+    branches: [main]
+
+jobs:
+  build-and-push-dev-aws:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3
+      - name: Get tag
+        run: echo "RELEASE_VERSION=${GITHUB_REF:10}" >> $GITHUB_ENV
+          
+      - name: Docker setup buildx
+        uses: docker/setup-buildx-action@4b4e9c3e2d4531116a6f8ba8e71fc6e2cb6e6c8c # v2
+        id: buildx
+        with:
+          install: true
+
+      - name: Build portal-client to e2e stage
+        uses: docker/build-push-action@3b5e8027fcad23fda98b2e3ac259d8d67585f671 # v4
+        with:
+          context: .
+          tags: portal-client:e2e
+          cache-to: type=local,mode=max,dest=/tmp/portal-client,tag=e2e
+          target: e2e
+
+      - name: Upload portal-client cache dir
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3
+        with:
+          name: portal-client
+          path: /tmp/portal-client
+
+      - name: Upload portal-client manifest
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3
+        with:
+          name: portal-client-index
+          path: /tmp/portal-client/index.json
+
+      - name: ls cache dirs
+        run: |
+          ls -l /tmp/portal-client
+          cat /tmp/portal-client/index.json | jq
+
+      - name: Set index.json file variable for extracting json
+        id: set_var
+        run: |
+          content=`cat /tmp/portal-client/index.json`
+          echo "index=$content" >> $GITHUB_OUTPUT
+
+      - name: Set sha digest of docker manifest
+        id: set_sha
+        env:
+          DIGEST: ${{fromJson(steps.set_var.outputs.index).manifests[0].digest}}
+        run: |
+          echo "sha256=${DIGEST##sha256:}" >> $GITHUB_OUTPUT
```

Comment:
Thought: Is the digest all we need from the `index.json`? If so would it be simpler to consolidate this step and the previous step?

Reply:
That's a good idea! I'll try to combine the steps.